Documentation/Makefile

```diff
@@ -1,3 +1,17 @@
+# Copyright 2017-2020 Authors of Cilium
```

Comment:
The copyright alternative line that @aanm was referring to is this:

```suggestion
# Copyright 2017-2020 Authors of Cilium
# SPDX-License-Identifier: Apache-2.0
```

It's human-readable, machine-readable, and refers the reader to the canonical Apache 2.0 license version which we should still have also at the top level of our repository.

Reply:
nit: I think SPDX is always on the first line, so this would need to be swapped. Would be great at some point to also change all the go code to SPDX. :)

Reply:
Happy to update these, so just two lines, right?

Reply:
yep.